student.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'Person'
+
+class Student < Person
+  def initialize(id, classroom:, name: 'Unknown', age: 0, parent_permission: true)
+    super(id, name: name, age: age, parent_permission: parent_permission)
+    @classroom = classroom
+  end
```

Comment:
- [ ] After creating the Person class you still need to ensure the classes that inherit from it are structured correctly. There is no `id`, `age` is not optional, the order of the parameters should be followed, and the way of assign the parameter should match the Person class.

  ```
  def initialize(classroom, age, name = 'Unknown', parent_permission: true)
    super(age, name, parent_permission: parent_permission)
    @classroom = classroom
  end
  ```